test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
I first added some extensive "randomized" tests which were only ran, if not in "short mode" (otherwise thy will be skipped): https://github.com/dedis/crypto/pull/86/files/9b4da9e2cc655606f348ec6b2c1394e90102d8ad?diff=unified#diff-287a11fd3cbb07e3e4f6b496c93fa64fR46
This isn't possible the way we do the tests now. @ineiti and me agreed to leave this tests if they don't really take much time.